esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -3313,6 +3313,31 @@ void getClusterConfig(char const * clusterType, char const * clusterName, char c
     return;
 }
 
+bool CWsWorkunitsEx::onWUGraphQuery(IEspContext &context,IEspWUGraphQueryRequest &req, IEspWUGraphQueryResponse &resp)
```

Comment:
I thought this was supposed to be a lightweight operation just to get the names. This gets the full graph information, which is not even remotely lightweight.